09-lambda/README.md

```diff
@@ -100,9 +100,10 @@ Use the AWS CLI to create Lambda functions:
   separate file and update the Lambda resource to reference the
   handler.
 
-- Use the "aws cloudformation package" and "\... deploy" commands to
-  create the CloudFormation stack. Note: The "package" command will
-  need an S3 bucket to temporarily store the deployment package.
+- Use the "aws cloudformation deploy" command to
```

Comment:
I think `aws cloudformation package` is still needed to be able to include the lambda code implementation as it's moved from inline in the instructions proceeding this.